src/Swarm/Language/Capability.hs

```diff
@@ -72,6 +72,10 @@ data Capability
     CSensehere
   | -- | Execute the 'Detect' command
     CDetectloc
+  | -- | Execute the 'Sniff' command
+    CDetectdistance
+  | -- | Execute the 'Chirp' command
+    CDetectdirection
```

Comment:
These capabilities don't have any devices that provide them, right?  Is the plan to leave that until later?

Reply:
Yes, will come up with those later.